software/gui/gui.sh

```diff
@@ -253,13 +231,15 @@ if [ "$1" == "help" ] || [ "$1" == "-h" ]; then
 elif [ "$1" == "install" ]; then
   if [ "$PLATFORM" == "Darwin" ]; then
     brew install qt5
+    configure_conan
     exit $EXIT_SUCCESS
   elif [ "$PLATFORM" == "Linux" ]; then
     if [ "$EUID" -ne 0 ]; then
       echo "Please run install with root privileges!"
       exit $EXIT_FAILURE
     fi
     install_linux
+    configure_conan
```

Comment:
Doing this with root privileges does not perform the required operations:
- if conan was installed as a per-user basis (which appears to be what happened to me when I ran `pip install conan` manually, as a side note, it would be nice to have `gui.sh install` install python packages as well), we get `unknown command: conan` error (`root` doesn't have conan.py in its path)
- otherwise, conan configuration would (probably) be installed in root's home instead of current user's home.

Reply:
Yes, turns out you should install it with sudo. I have changed the `gui.sh` script to do this.